esp_encrypted_img/README.md

```diff
@@ -0,0 +1,10 @@
+# Encrypted Image Decryptor component
+
+This component is used for decryption of encrypted images used during OTA updates.
+
+This component also contains tool to generate encrypted image which can be used during OTA updates.
+
+This component is used as part of following ESP-IDF examples:
```

Comment:
We can remove this line. We can keep reference to this component from IDF example README as that will be starting point for users.